packages/aws-cdk-lib/aws-s3/README.md

```diff
@@ -782,3 +782,89 @@ new s3.Bucket(this, 'Bucket2', {
   objectLockDefaultRetention: s3.ObjectLockRetention.compliance(Duration.days(365)),
 });
 ```
+
+## Replicating Objects
+
+You can use [replicating objects](https://docs.aws.amazon.com/AmazonS3/latest/userguide/replication.html) to enable automatic, asynchronous copying of objects across Amazon S3 buckets.
+Buckets that are configured for object replication can be owned by the same AWS account or by different accounts.
+You can replicate objects to a single destination bucket or to multiple destination buckets.
+The destination buckets can be in different AWS Regions or within the same Region as the source bucket.
+
+To replicate objects to a destination bucket, you can specify the `replicationRules` property:
+
+```ts
+declare const destinationBucket1: s3.IBucket;
+declare const destinationBucket2: s3.IBucket;
+declare const kmsKey: kms.IKey;
+
+const sourceBucket = new s3.Bucket(this, 'SourceBucket', {
+  // Versioning must be enabled on both the source and destination bucket
+  versioned: true,
+  replicationRules: [
+    {
+      // The destination bucket for the replication rule.
+      destination: s3.ReplicationDestination.sameAccount(destinationBucket1),
+      // The priority of the rule.
+      // Amazon S3 will attempt to replicate objects according to all replication rules.
+      // However, if there are two or more rules with the same destination bucket, then objects will be replicated according to the rule with the highest priority.
+      // The higher the number, the higher the priority.
+      // It is essential to specify priority explicitly when the replication configuration has multiple rules.
+      priority: 1,
+    },
+    {
+      destination: s3.ReplicationDestination.sameAccount(destinationBucket2),
+      priority: 2,
+      // Whether to specify S3 Replication Time Control (S3 RTC).
```

Comment:
Please spare a sentence to explain what enabling RTC means.

According to the docs it has something to do with metrics, but metrics is a separate field here, so now I don't understand what this does.

Reply:
I've updated like this.

```ts
      // Whether to specify S3 Replication Time Control (S3 RTC).
      // S3 RTC replicates most objects that you upload to Amazon S3 in seconds,
      // and 99.99 percent of those objects within specified time.
      replicationTimeControl: s3.ReplicationTimeValue.FIFTEEN_MINUTES,
      // Whether to enable replication metrics about S3 RTC.
      // If set, metrics will be output to indicate whether replication by S3 RTC took longer than the configured time.
      metrics: s3.ReplicationTimeValue.FIFTEEN_MINUTES,
```